### ISSUE TITLE
[Enhancement]Add Hand & Metadata

### ISSUE THREAD

**Issue by damonmensch** [hsyyid/Kits#1]
When adding an item to a kit perhaps add the optional use of "add hand", or make a new subcommand  to do the same. This will allow users to add whatever is currently in their hand.

Also maybe allow the ability to allow kits to recognize/distribute data via the metadata tags. This would add additional compatibility to mods such as Tinker's Construct which use the same ID for multiple things and tags are used instead to differentiate. The above suggestion would assist in this functionality when adding items to the kits.

**Comment by hsyyid** [comment]
Hello. I am currently waiting for the Inventory portion of the SpongeAPI to be implemented, which will eradicate the need for almost all commands. It will allow users to make whatever is in their inventory into a kit easily - making it so users will not need an items id, etc. Also, metadata is configured so you can add an item with specific metadata. It is listed on the Sponge page for Kits.

**Comment by damonmensch** [comment]
Guess I should of been more specific, I was actually referring to the NBT tags, which are essentially metadata, not just damage values.

**Comment by hsyyid** [comment]
These? http://minecraft.gamepedia.com/Tutorials/Command_NBT_Tags

**Comment by damonmensch** [comment]
Yes those, many mods use them for various reasons, Tinker's for which part has what material type, TE2 for charges of batteries/blocks.

**Comment by hsyyid** [comment]
I'll see.
